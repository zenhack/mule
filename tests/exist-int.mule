#XFAIL: type formatting
4 : exist t. t
