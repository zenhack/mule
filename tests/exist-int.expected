
Desugared:

	(: (Integer 4) (exist t t))

inferred type:

	(exist a a)

Runtime term:

	(Const (Integer 4))

Evaluated:

	(Const (Integer 4))
